.github/workflows/test.yml

```diff
@@ -643,6 +643,10 @@ jobs:
             path: subworkflows/nf-core/fastq_align_bamcmp_bwa
           - profile: conda
             path: subworkflows/nf-core/fastq_align_bwa
+          - profile: conda
+            path: modules/neuroimaging
+          - profile: singularity
+            path: modules/neuroimaging
```

Comment:
Why skip singularity?

Reply:
For now, the path to the singularity is unstable on our end. I even think 2.0.2 is not available now. A skip until I resolve some things at my lab.

Reply:
Okay this should really be fixed before the pipeline release indeed. A lot of systems depend on singularity for their pipeline runs